2024/calcite-react/demo/README.md

```diff
@@ -0,0 +1,25 @@
+# 2024 Esri Developer Summit: Building Web Apps with Calcite Design System and React (Demo App)
+
+Description: TODO
+
+## Technologies used:
+- [Calcite Design System](https://developers.arcgis.com/calcite-design-system/)
+- [ArcGIS Maps SDK for JavaScript's ES modules](https://developers.arcgis.com/javascript/latest/)
+- [React](https://react.dev/)
+- [TypeScript](https://www.typescriptlang.org/)
+- [Vite](https://vitejs.dev/)
+
+## Steps to run web app locally, in terminal, run:
+
+1. `git clone https://github.com/maxpatiiuk/esri-dev-summit-presentations.git`
+2. `cd esri-dev-summit-presentations/2024/calcite-react/demo`
+3. `yarn install`
```

Comment:
I'm not against yarn, but more people have `npm` installed by default
Should we use `npm install` and `npm start` here instead? They should work equivalently in this case

Reply:
Good point, updated yarn to npm